src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/SlayerFeatures.kt

```diff
@@ -249,9 +249,9 @@ object SlayerFeatures : CoroutineScope {
         hasSlayerText = index != -1
         if (!lastTickHasSlayerText && hasSlayerText) {
             sidebarLines.elementAtOrNull(index - 1)?.let {
-                val boss = it.substringBeforeLast(" ")
```

Comment:
Does this substring the same way it used to? Looks like one was removed.

Reply:
Yes, it grabs the first word from the boss name and properly gets the expected hp, just without the redundant variable assignment. Compared to main branch, the system is used for the "Use Hits to Detect Slayer" feature, but it now works for all slayers instead of only Voidgloom.